.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: ruby
+cache: bundler
```

Comment:
What does this do?

Reply:
https://docs.travis-ci.com/user/caching/#Caching-directories-(Bundler%2C-dependencies)